src/Compilers/CSharp/Test/Emit/CodeGen/SwitchTests.cs

```diff
@@ -114,12 +114,12 @@ public class Test
 {
     public static int Main(string [] args)
     {
-		int ret = 1;
-		switch (true) {
-		case true:
-			ret = 0;
-			break;
-		}
+        int ret = 1;
+        switch (true) {
+        case true:
+            ret = 0;
+            break;
+        }
```

Comment:
Because the source file had a mixture of spaces and tabs, and I turned them uniformly into spaces.